.github/workflows/check_for_broken_links.yml

```diff
@@ -1,5 +1,8 @@
 name: LinkChecker
-on: workflow_dispatch
+on:
+  schedule:
+    - cron: '0 17 * * *'
```

Comment:
Should this run on weekdays only? So, `0 17 * * 1-5`?

https://crontab.guru/#0_17_*_*_1-5